src/components/Login/Login.js

```diff
@@ -3,24 +3,54 @@ import logo from './ebuddy-logo.svg';
 import grafik from './grafik-login.svg';
 import papierflieger from './Papierflieger.svg';
 
-function Login() {
+function Login({ onLogin }) {
+
+    const userData = [
+        {
+            username: 'FabianHanso',
+            password: '123',
+        },
+        {
+            username: 'JohnDoe',
+            password: '456',
+        }
+    ]
+
   return (
     <Wrapper>
         <TopWrapper>
             <LoginLogo src={logo} alt="Logo" />
             <LoginGrafik src={grafik} alt="Schulbücher" />
         </TopWrapper>
         <BottomWrapper>
-            <Form>
-                <InputField type="text" placeholder="Vorname" />
-                <InputField type="text" placeholder="Passwort" />
-                <InputButton type="submit">Anmelden</InputButton>
+            <Form onSubmit={FormSubmitHandler}>
+                <InputField type="text" placeholder="Vorname" required name="username"></InputField>
+                <InputField type="Password" placeholder="Passwort" required name="password"></InputField>
```

Comment:
**_Please_** add `label` elements. I marked this in your last PR already, please make sure to add them.